std_msgs/Header header
uint32 sending_agent
uint8[] data
DetectedObjectWithImage[] objects